Description: do not fail when D-Bus is not available on configure time
 According to comments in https://codereview.qt-project.org/125499, this check
 will be dropped before the final Qt 5.6 release.
Author: Dmitry Shachnev <mitya57@debian.org>
Last-Update: 2015-12-29

--- a/tests/auto/auto.pro
+++ b/tests/auto/auto.pro
@@ -33,10 +33,6 @@
 # Disable the QtDBus tests if we can't connect to the session bus
 qtHaveModule(dbus) {
     !system("dbus-send --session --type=signal / local.AutotestCheck.Hello >/dev/null 2>&1") {
-        contains(QT_CONFIG, dbus-linked): \
-            error("QtDBus is enabled but session bus is not available. Please check the installation.")
-        else: \
-            warning("QtDBus is enabled with runtime support, but session bus is not available. Skipping QtDBus tests.")
         SUBDIRS -= dbus
     }
 } else {
